steps/ionic-prepare/1.2.0/step.yml

```diff
@@ -0,0 +1,86 @@
+title: Ionic prepare
+summary: Prepares your Ionic project
+description: |-
+  Prepares your Ionic project.
+
+  The step prepares your Ionic project by calling:
+
+  - `ionic cordova "prepare" "--no-build"`
+website: https://github.com/bitrise-steplib/steps-ionic-prepare
+source_code_url: https://github.com/bitrise-steplib/steps-ionic-prepare
+support_url: https://github.com/bitrise-steplib/steps-ionic-prepare/issues
+published_at: 2020-11-09T16:39:27.754652877Z
+source:
+  git: https://github.com/bitrise-steplib/steps-ionic-prepare.git
+  commit: 6af9c291bba9d5f9ddb14170d9afe435f42a50ab
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ionic
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-ionic-prepare
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
```

Comment:
Could you please refactor the inputs' of the step? Could you please provide a summary of them?
```
Summary should be shorter than description, but there are cases that they are the same. It should not be empty.
```

Example:
https://github.com/bitrise-io/bitrise-steplib/pull/2720/files#diff-0d7d7d44d621f57e0d2dae0159715f1719e3761937be7f3eb298f56ac0b9ca62R45

Reply:
Fixed.